src/OpenTelemetry.AutoInstrumentation/Configurations/FailFastSettings.cs

```diff
@@ -0,0 +1,27 @@
+// <copyright file="FailFastSettings.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+namespace OpenTelemetry.AutoInstrumentation.Configurations;
+
+internal class FailFastSettings : Settings
+{
+    public bool FailFast { get; private set; }
+
+    protected override void OnLoad(Configuration configuration)
+    {
+        FailFast = configuration.GetBool(ConfigurationKeys.FailFast) ?? false;
```

Comment:
😄 can we make fail fast fail fast if fail fast fail?

Reply:
:D no!